drud/ddev
# Add PHP version information to ddev describe

## The Problem/Issue/Bug:

Users need to be able to see what version of PHP a given site is using.

## How this PR Solves The Problem:

PHP version info is now reported in the output of `ddev describe`. We're only showing detail down to the minor version (e.g. `5.6`, `7.0`, `7.1`) because more info than that isn't actionable by the user from the ddev cli. More advanced users that need specifics can use `ddev exec` to get the output of `php --version`. Ran this by @rickmanelius and he gave the approach a :+1:.

## Manual Testing Instructions:

Run `ddev describe` and see the new PHP version row in the output. This didn't really fit into the other categories, so I created a new one. We can add more stuff to it in the future if needed.

## Automated Testing Overview:
The test for `ddev describe` now checks the PHP version output as well.

## Related Issue Link(s):
https://github.com/drud/ddev/issues/487

## Release/Deployment notes:
